<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>yuuri_bulge.gif</normaloff>yuuri_bulge.gif</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(81, 0, 135, 255), stop:0.427447 rgba(41, 61, 132, 235), stop:1 rgba(155, 79, 165, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="reloadRulesBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255, 255, 255, 30);
color: white;
font-size: 16px;
}

QPushButton:pressed {
background-color: rgba(255, 255, 255, 40);
}</string>
    </property>
    <property name="text">
     <string>Reload rules</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>1000</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
background-color: rgba(255, 255, 255, 30);
color: black;
}

QTableWidget::item {
border-style: none;
}
</string>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Col1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Col2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Col3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Col4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Col5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Col6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Col7</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Col8</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Col9</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="addRuleBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255, 255, 255, 30);
color: white;
font-size: 16px;
}

QPushButton:pressed {
background-color: rgba(255, 255, 255, 40);
}</string>
    </property>
    <property name="text">
     <string>Add rule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteRuleBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255, 255, 255, 30);
color: white;
font-size: 16px;
}

QPushButton:pressed {
background-color: rgba(255, 255, 255, 40);
}</string>
    </property>
    <property name="text">
     <string>Delete rule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startFwcBtn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255, 0, 0, 128);
color: white;
font-size: 16px;
border-radius: 50%;
}</string>
    </property>
    <property name="text">
     <string>Start the FW</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
